api/working-with-extensions/publishing-extension.md

```diff
@@ -312,7 +312,7 @@ We only support `major.minor.patch` for extension versions and `semver` pre-rele
 VS Code will auto update extensions to the highest version available, so even if a user opted into a pre-release version and there is an extension release with a higher version, that user will be updated to the released version.
 Because of this we recommend that extensions use `major.EVEN_NUMBER.patch` for release versions and `major.ODD_NUMBER.patch` for pre-release versions. For example: `0.2.*` for release and `0.3.*` for pre-release.
 
-If extension authors do not want their pre-release users to be updated to the release version, we recommend to always increment and publish a new pre-release version before publishing a release version in order to make sure that the pre-release version is always higher.
+If extension authors do not want their pre-release users to be updated to the release version, we recommend to always increment and publish a new pre-release version before publishing a release version in order to make sure that the pre-release version is always higher. Note that while pre-release users will be updated to a release version if it is newer, they still remain eligible to automatically update to future pre-releases with higher version numbers than the release version.
```

Comment:
Good clarification.
Can you just instead of says "if it is newer" say "if it is higher" that way it sounds more precise to me.

@sandy081 fyi

Reply:
Updated.